src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -37,6 +37,7 @@
             .withPhone("85355255")
             .withWebsite(Website.WEBSITE_EXAMPLE)
             .withTags("friends")
+            .withPicture(null)
```

Comment:
in the future update we will probably have to add a sample picture just to make sure alls good